packages/webview_flutter/webview_flutter_android/example/lib/main.dart

```diff
@@ -33,6 +33,19 @@ The navigation delegate is set to block navigation to the youtube website.
 </html>
 ''';
 
+const String kVideoExamplePage = '''
```

Comment:
I have updated the example to load the "Welcome to Flutter" video on Youtube (https://www.youtube.com/watch?v=4AoFA19gbLo). 

Some extra information on the change in the `onNavigationRequest` callback:
On my initial try Youtube loaded fine, even though the `onNavigationRequest` callback prevented from navigating to sites starting with `https://www.youtube.com/`. After some debugging I realised that Youtube actually redirects to `https://m.youtube.com`. So I decided to update the logic in the example app to prevent loading websites with URL containing `youtube.com`. This prevented loading the Youtube page correctly. Since we do actually want to load the Youtube site, I have next updated the criteria to prevent loading websites with the URL containing the string `pub.dev`.